lib/Git/Hooks.pm

```diff
@@ -1093,7 +1093,9 @@ sign, you can configure it like this:
       'eval:(exists $ENV{GERRIT_USER_EMAIL} && $ENV{GERRIT_USER_EMAIL} =~ /([^@]+)/) ? $1 : undef'
 
 This variable is useful for any hook that need to authenticate the
-user performing the git action.
+user performing the git action. The value is available in CheckReference hook
+when specifying allowed reference (branch) names. It can be access
+by string '{Git::Hooks-Auth-User}' (without single quotes).
```

Comment:
I think it would be better to place this on the documentation for the Git::Repository::Plugin::GitHooks::grok_acls function.

Reply:
Added there.